rts/motoko-rts/src/gc/copying.rs

```diff
@@ -0,0 +1,242 @@
+use crate::alloc;
```

Comment:
I assume the code in here has just moved, and isn't substantially changed right?

Reply:
Yes, no need to review this file.

Reply:
This file is now changed with 9af5279.